oeps/oep-0028-content-theming-in-react.rst

```diff
@@ -0,0 +1,206 @@
+==================================
+OEP-0028: Content Theming in React
+==================================
+
+.. This OEP template is based on Nygard's Architecture Decision Records.
+
++-----------------+----------------------------------------------------------------+
+| OEP             | :doc:`OEP-0028 </oeps/oep-0028-arch-content-theming-react>`    |
+|                 |                                                                |
+|                 |                                                                |
+|                 |                                                                |
+|                 |                                                                |
++-----------------+----------------------------------------------------------------+
+| Title           | Content Theming in React                                       |
++-----------------+----------------------------------------------------------------+
+| Last Modified   | 2018-08-17                                                     |
++-----------------+----------------------------------------------------------------+
+| Authors         | Braden MacDonald                                               |
++-----------------+----------------------------------------------------------------+
+| Arbiter         |                                                                |
++-----------------+----------------------------------------------------------------+
+| Status          | Draft                                                          |
++-----------------+----------------------------------------------------------------+
+| Type            | Architecture                                                   |
++-----------------+----------------------------------------------------------------+
+| Created         | 2018-08-17                                                     |
++-----------------+----------------------------------------------------------------+
+| `Review Period` | 2018-08-17 - 2018-08-25                                        |
++-----------------+----------------------------------------------------------------+
+
+Context
+-------
+
+This section describes the forces at play, including technological, political,
+social, and project local. These forces are probably in tension, and should
+be called out as such. The language in this section is value-neutral. It is
+simply describing facts.
```

Comment:
I think this is something you can perhaps post in the PR description, but doesn't really make sense in the OEP itself, since that is a long-term document.

Reply:
Ah! OK.